src/Features/Traits/Console.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Illuminate\Support\Facades\Artisan;
+use PHPUnit\Framework\Assert;
+
+trait Console
+{
+    use Application;
+
+    private string $consoleOutput = "";
+
+    /**
+     * @given I run shell command :command
+     * @given I run shell command :command in console
+     */
+    public function runShellCommand(string $command): void
+    {
+        $this->consoleOutput = "";
+        exec($command, $output);
+        $this->consoleOutput = implode("\n", $output);
+    }
+
+    /**
+     * @Given I run artisan command :command
+     * @Given I run artisan command :command in console
+     */
+    public function runArtisanCommand(string $command): void
+    {
+        $this->consoleOutput = "";
+        Artisan::call($command);
```

Comment:
Please get artisan console from the dependency container. We should not use facades in packages as described here: https://laravel.com/docs/11.x/packages#a-note-on-facades